src/stores.ts

```diff
@@ -72,3 +74,27 @@ export const regionalTrends = writable(rt);
 // This helps prevent repeatedly fetching the large file.
 export const isZipsDownloaded = writable(false);
 
+/** 
+ * A store to contain an array of all valid dates
+ */
+export const dateRange = writable<string[]>(fetchDateData());
+/**
+ * A method to read in the ALL_dates.csv file into an array of dates strings.
+ * @returns an array of date string values
+ */
+ function fetchDateData() {
```

Comment:
data fetching and parsing functions should live in data.ts.

Reply:
moved this block of code to data.ts.